_sass/_blog.scss

```diff
@@ -1,27 +1,28 @@
 .blog-sidebar {
-  background-color: $navy-drk;
-  border-radius: 4px;
   height: 100%;
   width: 300px;
   margin: {
     top: 35px;
     bottom: 48px;
     right: 40px;
   }
+  background-color: $navy-drk;
+  @include border-radius(4px);
+  @include media_max(1030px) { display: none; }
```

Comment:
I don't think we should hide this entirely on mobile. We should just move it to the bottom on mobile.